packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuideButton/FieldGuideButton.js

```diff
@@ -1,18 +1,39 @@
 import { SpacedText } from '@zooniverse/react-components'
 import { Button, Box } from 'grommet'
 import PropTypes from 'prop-types'
-import styled, { css, withTheme } from 'styled-components'
```

Comment:
`withTheme` is not needed here. `theme` is automatically available in `styled` components.